docs/react/README.md

```diff
@@ -1,72 +1,186 @@
 # @hatchifyjs/react
 
-- [What is @hatchifyjs/react](#what-is-hatchifyjsreact)
-  - [TypeScript](#typescript)
-  - [Types](#types)
+@hatchifyjs/react is a schema-driven library of components and helper methods for a Hatchify + React frontend. By providing the schemas (AKA models) of your backend resources, @hatchifyjs/react returns a set of components and helper functionality that you can use across your React app.
+
+`@hatchifyjs/react` is a TypeScript supported [NPM package](https://www.npmjs.com/package/@hatchifyjs/react) that takes your [JSON:API rest client](#createjsonapiclient) and[Schema(s)](../schema/README.md) to produce custom:
```

Comment:
i'd word it along the lines of:

`@hatchifyjs/react` is an [NPM package](https://www.npmjs.com/package/@hatchifyjs/react) that takes your schemas and provides:

- Components
- CRUD promise functions and react hooks
- TypeScript